modules/acquia_cms_common/src/EventSubscriber/KernelTerminate/AcquiaConnectorTelemetryOverride.php

```diff
@@ -26,8 +28,106 @@ public function onTerminateResponse(KernelEvent $event) {
     // Check if telemetry opted,
     // then only trigger AcquiaTelemetry::onTerminateResponse($event).
     if ($this->state->get('acquia_connector.telemetry.opted', TRUE)) {
+      if ($this->state->get('acquia_cms_telemetry.telemetry') === NULL) {
+        $eventType = 'ACMS Telemetry data';
+        $this->sendAcmsTelemetry($eventType);
+      }
       parent::onTerminateResponse($event);
     }
   }
 
+  /**
+   * Sends an event to Amplitude.
+   *
+   * @param array $event
+   *   The Amplitude event.
+   *
+   * @throws \GuzzleHttp\Exception\GuzzleException
+   *
+   * @see https://developers.amplitude.com/#http-api
+   */
+  private function sendAcmsEvent(array $event) {
+    $this->httpClient->request('POST', $this->apiUrl, [
+      'form_params' => [
+        'api_key' => 'e896d8a97a24013cee91e37a35bf7b0b',
```

Comment:
@deepakmishra2 we cannot send api_key like this.

cc: @vishalkhode1

Reply:
@chandan-singh7929 any comment on this?

Reply:
I don't see any issue using this.